newrelic/resource_newrelic_one_dashboard_json_test.go

```diff
@@ -161,6 +161,9 @@ func testAccCheckNewRelicOneDashboardJsonConfig_Full(pageName string, accountID
 					"query": "FROM Transaction SELECT average(duration) FACET appName"
 				},
 				"name": "variableNRQL",
+				"options": {
+					"ignoreTimeRange": true
+				},
```

Comment:
It might also be a good idea to update one of the tests in `newrelic_one_dashboard_test.go` with this new attribute